### PR TITLE
Created a struct defining playButtonState.

### PR DESCRIPTION
**Pull request by wokalski**
The previous solution was bad IMO because `PlayButton` "knew" about selectors in the main controller.

### CODE REVIEW

**Comment by wokalski** [comment]
@maartenschumacher

File: GameOfLife/PlayButton.swift [hunk 1]
```diff
@@ -8,22 +8,18 @@
...
-    
-    func setButtonState(state: ButtonState) {
-        self.setTitle(state.title, forState: UIControlState.Normal)
-        self.currentPlayState = state
-    }
-    
-}
+    var currentPlayState: PlayButtonState!
+    {
+        didSet {
```
Comment (maartenschumacher) [inline GameOfLife/PlayButton.swift:20]: Cool!

**Comment by maartenschumacher** [comment]
@wczekalski Great! Thanks :)

**Comment by wokalski** [comment]
I learn about Swift thanks to this; Thanks!

**Pull request merged**